### PR TITLE
Add promisified query mode

### PR DESCRIPTION
**Pull request by imatveev**

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -288,6 +288,17 @@ module.exports = class Client extends EventEmitter {
...
+  queryAsync (sql, opts) {
+    return new Promise((resolve, reject) => {
+      this.query(sql, opts, (error, result) => {
+        if (error) {
+          return reject(error);
+        }
+        resolve(result);
+      });
+    });
+  }
```
Comment (vweevers) [inline lib/client.js:300]: Can you fix the code style issues here? Run `npm test` to check, and `npx standard --fix` to automatically fix them.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lento",
-  "version": "2.3.0",
+  "version": "2.3.1",
```
Comment (vweevers) [inline package.json]: Please revert this, the version should not be bumped in pull requests.

File: readme.md [hunk 3]
```diff
@@ -172,8 +173,18 @@ client.query('DESCRIBE events', (err, rows) => {
...
 
-I'll take a PR for Promise support.
+Same as above but promise-based.
+
+```js
+try {
+  const rows = await client.queryAsync('DESCRIBE events');
+catch(error) {
+  // ..
+}
```
Comment (vweevers) [inline readme.md]:
```suggestion
const rows = await client.queryAsync('DESCRIBE events')
```

Adding `try/catch` to the example is not necessary.

Comment (imatveev) [reply readme.md]: Done

File: package.json (hunk 2 above)
Comment (imatveev) [reply package.json]: Done

File: lib/client.js (hunk 1 above)
Comment (imatveev) [reply lib/client.js:300]: Done

**Comment by imatveev** [comment]
@vweevers Went over comments, please review when you have a time

**Comment by vweevers** [comment]
I went a slightly different way and added promise support to all methods (that didn't have a return value): https://github.com/vweevers/lento/commit/11d6c2b0ff332418210d3df4a50af0a3e3414c4e. For example:

```js
const rows = await client.query('DESCRIBE events')
```

Released in 2.4.0.

Thanks for the PR!

**Pull request closed without merging**